### PR TITLE
Aruiz/feat/86b2uqajp/add mui to boilerplate

### PR DESCRIPTION
**Pull request by Ralexs0096**
**Hello Team:**

### Description
This Pull Request addresses the following updates and improvements:

- Remove the default configuration of tailwind in favor of using MUI.
- Add MUI and its dependencies (roboto font)
- Create an initial component using MUI Components. 

- **Testing:**
N/A

Please review the changes at your convenience. Your feedback would be greatly appreciated!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,7 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+
+# enviromental variable
+.env
```
Comment (AlvisonHunterArnuero) [inline .gitignore]:
```suggestion
.env
.env.local
.env.*.local
```

**Review by AlvisonHunterArnuero — CHANGES_REQUESTED** [review]
nit, add these as well if seen fit, please.

File: .gitignore (hunk 1 above)
Comment (Ralexs0096) [reply .gitignore]: applied

**Review by AlvisonHunterArnuero — APPROVED** [review]
LGTM!

**Pull request merged**